gfsphysics/GFS_layer/GFS_typedefs.F90

```diff
@@ -3059,6 +3065,9 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
 
     logical              :: use_ufo        = .false.         !< flag for gcycle surface option
 
+!--- flake model parameters
+    integer              :: lkm            =  1  !< flag for flake model
```

Comment:
Why is this flag declared as integer? What possible values can it take?

Reply:
If this is changed to a logical (which makes sense if it can be either .false. = 0 or .true. = 1, but not if you anticipate having other options in the future (0,1,2,...) ), then we need changes in ccpp-physics as well.